### PR TITLE
Feature/issue templates

### PR DESCRIPTION
**Pull request by DavidNic11**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+description: I experienced unexpected behavior using useSimpleReducer.
```
Comment (christopherjbaker) [inline .github/ISSUE_TEMPLATE/bug.yml]: I would remove anything project-specific from these templates so we can just copy-paset them between projects.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+        What environments were you in when you encoutnered this bug?
+
+  - type: input
+    attributes:
+      label: Node
+      description: What version of Node?
```
Comment (christopherjbaker) [inline .github/ISSUE_TEMPLATE/bug.yml:18]: and NPM

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      required: true
+
+  - type: input
+    attributes:
+      label: React
+      description: What version of React?
+      placeholder: '17'
+    validations:
+      required: true
+
```
Comment (christopherjbaker) [inline .github/ISSUE_TEMPLATE/bug.yml:38]: What meta-frameworks and build systems are you using, and their versions? NextJS, Gatsby, Vite, etc.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        ## Reproduction
+
+  - type: input
+    attributes:
+      label: CodeSandbox
+      description: To expedite the process, provide a link to a CodeSandbox or Repository where this error can be reproduced. You can use [Bitovi's useSimpleReducer Template](https://codesandbox.io/s/github/bitovi/use-simple-reducer/tree/error-handling/demo) to create one quickly.
```
Comment (christopherjbaker) [inline .github/ISSUE_TEMPLATE/bug.yml]: specifically, we want _minimal_ reproduction sandboxes. We don't want somebody to stuff their project in, we want them to make the smallest possible app that reproduces it.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+> **Before You Begin**
+> If you have not already please read through the contributing guidelines
+
+## Checklist before requesting a review
+
+- [ ] I have followed the guidelines in the [Contributing document](../CONTRIBUTING.md)
+- [ ] I checked to make sure there aren't other open [Pull Requests](https://github.com/bitovi/use-simple-reducer/pulls) for the same update/change?
```
Comment (christopherjbaker) [inline .github/pull_request_template.md]: I wonder if we could make a relative link work here? (so that there's nothing project-specific in the file)

**Review by christopherjbaker — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 5 above)
Comment (DavidNic11) [reply .github/pull_request_template.md]: Oh interesting. I've never tried that on an md file. I'll try it real quick

**Review by christopherjbaker — APPROVED** [review]

**Pull request merged**